src/Coduo/PHPHumanizer/String/HtmlTruncate.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace Coduo\PHPHumanizer\String;
+
+class HtmlTruncate extends TextTruncate
+{
+    /**
+     * @var string
+     */
+    private $text;
+
+    /**
+     * @var int
+     */
+    private $charactersCount;
+
+    /**
+     * @var string
+     */
+    private $append;
+
+    /**
+     * @var string
+     */
+    private $allowedTags;
+
+    /**
+     * @param string $text
+     * @param int    $charactersCount
+     * @param string $allowedTags
+     * @param string $append
+     */
+    public function __construct($text, $charactersCount, $allowedTags = '<b><i><u><em><strong><a><span>', $append = '')
+    {
+        $this->text            = $text;
+        $this->charactersCount = $charactersCount;
+        $this->append          = $append;
+        $this->allowedTags     = $allowedTags;
+    }
+
+    /**
+     * Truncates a string to the given length.  It will optionally preserve
+     * HTML tags if $is_html is set to true.
+     *
+     * Adapted from FuelPHP Str::truncate (https://github.com/fuelphp/common/blob/master/src/Str.php)
+     *
+     * @param string $string
+     *
+     * @return  string  the truncated string
+     */
+    private function truncateHtml($string)
```

Comment:
Please move private functions under the protected and public. 
This is the order of methods and fields in classes I would like to follow:
```
class
{
    public $field;
    protected $field;
    private $field;

    public function() {}
    protected function() {}
    private function() {}
}
```